### ISSUE TITLE
As a SO/SE I want to create a digital flyer that can be printed with background images and linked text

### ISSUE THREAD

**Issue by ekellagher** [CuBoulder/express#2293]
A client requested an webpage option for creating flyers.
https://cuboulder.atlassian.net/browse/EXP-9592

I thought the section bundle was a nice solution with its unique navigation option but it does not print well
Site with section bundle enabled
www.colorado.edu/safety

**Comment by kevincrafts** [comment]
I don't think a website building tool is the best tool to use to create flyers.  We are dealing with much different resolutions and so the appropriate tool (Adobe InDesign, etc.)should be used.
